ui/src/app/components/popups.tsx

```diff
@@ -242,11 +244,20 @@ export const Popup = fp.flow(
   }
 });
 
-export class PopupTrigger extends React.Component {
+interface PopupTriggerProps {
```

Comment:
IIRC we started a norm of naming these "Props" and only getting more specific on export

Reply:
That makes sense, however, this is a file with many different components, and so there would be name collisions. I am not sure I would consider PopupTrigger the 'main' component of this this file? (it's a little less clear in shared component files)

Reply:
OK - let's keep it like this